File: Server/src/main/java/com/api/server/foodb/HealthEffect.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "health_effects")
public class HealthEffect {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Size(max = 255)
    @NotNull
    @Column(name = "name", nullable = false)
    private String name;

    @Lob
    @Column(name = "description")
    private String description;

    @Size(max = 255)
    @Column(name = "chebi_name")
    private String chebiName;

    @Size(max = 255)
    @Column(name = "chebi_id")
    private String chebiId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

    @Column(name = "creator_id")
    private Integer creatorId;

    @Column(name = "updater_id")
    private Integer updaterId;

    @Lob
    @Column(name = "chebi_definition")
    private String chebiDefinition;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/HealthEffect.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("health_effects")]
[Index("ChebiId", Name = "index_health_effects_on_chebi_id")]
[Index("ChebiName", Name = "index_health_effects_on_chebi_name")]
[Index("Name", Name = "index_health_effects_on_name", IsUnique = true)]
public class HealthEffect
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("name")] public string Name { get; set; } = null!;

    [Column("description", TypeName = "mediumtext")]
    public string? Description { get; set; }

    [Column("chebi_name")] public string? ChebiName { get; set; }

    [Column("chebi_id")] public string? ChebiId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }

    [Column("creator_id")] public int? CreatorId { get; set; }

    [Column("updater_id")] public int? UpdaterId { get; set; }

    [Column("chebi_definition", TypeName = "text")]
    public string? ChebiDefinition { get; set; }
}